gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -123,7 +124,7 @@ protected Path computeNext() {
    */
   @VisibleForTesting
   public static void setGCloudOptions(StorageOptions newStorageOptions) {
-    storageOptions = newStorageOptions;
+    futureStorageOptions = newStorageOptions;
```

Comment:
Given that we are touching this method, can you rename it to `setStorageOptions()`? We are dropping the the `gcloud` naming soon.